app/views/TaxYearSummary.scala.html

```diff
@@ -368,6 +368,7 @@ <h1 class="govuk-heading-xl" id="heading">
                 )
             )))
         }
+        @if(!errorPresent(model)) {
```

Comment:
@AshokGitAccount  - I would assume this was not part of the IT test originally (was not covered ) before we started working on this story?

Reply:
can you add IT test to cover this scenario ?

Reply:
@avelx - added check for forecast table to not be displayed if there are error messages in the existing IT

Reply:
@AshokGitAccount - do we need to update spec for agent as well?